client/components/date-range/test/index.js

```diff
@@ -115,11 +118,17 @@ describe( 'DateRange', () => {
 				/>
 			);
 
+			let [ actualStartDate, actualEndDate ] = screen
```

Comment:
This gets repetitive, I wonder if we could abstract it to a helper function.

Reply:
That's a good point 👍🏻 I'll do it.

Reply:
See https://github.com/Automattic/wp-calypso/pull/63879#discussion_r906322168.